.github/workflows/cmake.yml

```diff
@@ -546,6 +561,12 @@ jobs:
             gcov-exec: gcov-10
             codecov: macos_gcc
 
+          - name: macOS Clang Native Instructions (ARM64)
+            os: macos-14
```

Comment:
Can we use `macos-latest` here instead? If there is a reason, perhaps it should be commented?

Reply:
https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners/about-github-hosted-runners

> The macos-latest workflow label currently uses the macOS 12 runner image.

Therefore we need macos-14 (first ARM64 runner).

Reply:
I see. Got it.